h/search/core.py

```diff
@@ -152,3 +152,10 @@ def _parse_aggregation_results(self, aggregations):
         for agg in self._aggregations:
             results[agg.name] = agg.parse_result(aggregations)
         return results
+
+    @staticmethod
+    def _get_total_hits(response):
+        total = response["hits"]["total"]
+        if isinstance(total, int):
+            return total  # ES 6.x
+        return total["value"]  # ES 7.x
```

Comment:
I tacked this onto the class rather than have it separate. If you move the class you want this to come with.